content/resources/mat102.md

```diff
@@ -13,10 +13,26 @@ Welcome to your first proof-based mathematics course at UofT!
 
 Why is Math Proof important?
 
-- A proof, mathematical or not, is an argument or explanation that shows that something is absolutely, unequivocally true, beyond any shadow of a doubt. When you have accomplished that, it feels like magic.
+- A proof, mathematical or not, is an argument or explanation that shows that
+  something is absolutely, unequivocally true, beyond any shadow of a doubt.
+  When you have accomplished that, it feels like magic.
 
 - The satisfaction when you write down Q.E.D.
```

Comment:
Although this is true, I don't think we'll need this lol

Reply:
I think we should leave it.